fiftyone/utils/fiw.py

```
@@ -283,51 +286,331 @@ def _parse_identifier(identifier):
+        split: the split to prepare
+
+    Returns:
+        a tuple of (num_samples, classes)
+    """
+    _validate_source_dir(source_dir)
+    etau.ensure_dir(dataset_dir)
+    _organize_fiw_data(source_dir, dataset_dir)
+    return _get_dataset_info(dataset_dir, split)
+
+
+def download_fiw_dataset(dataset_dir, split, source_dir=None, scratch_dir=None, cleanup=False):
```

Ruff (0.14.8)

Boolean default positional argument in function definition

(FBT002)